src/main/resources/db/migration/V4_7__create_table_motedeltaker_arbeidsgiver_varsel.sql

```diff
@@ -0,0 +1,21 @@
+-- ROLLBACK-START
+------------------
+-- DROP TABLE MOTEDELTAKER_ARBEIDSGIVER_VARSEL;
+-- DROP INDEX IX_MOTEDELTAKER_ARBEIDSGIVER_VARSEL_MD_AT_ID;
+---------------
+-- ROLLBACK-END
+
+CREATE TABLE MOTEDELTAKER_ARBEIDSGIVER_VARSEL
+(
+    id                           SERIAL PRIMARY KEY,
+    uuid                         VARCHAR(50)  NOT NULL UNIQUE,
+    created_at                   TIMESTAMP    NOT NULL,
+    updated_at                   TIMESTAMP    NOT NULL,
+    motedeltaker_arbeidsgiver_id INTEGER REFERENCES MOTEDELTAKER_ARBEIDSGIVER (id) ON DELETE CASCADE,
+    varseltype                   VARCHAR(30)  NOT NULL,
+    pdf                          bytea        NOT NULL,
+    status                       VARCHAR(100) NOT NULL,
+    lest_dato                    TIMESTAMP
```

Comment:
Fint om du kan legge til:
fritekst TEXT NOT NULL DEFAULT '',
document JSONB NOT NULL DEFAULT '[]'::jsonb

Reply:
Ja stemmer. glemte å ta med det som var blitt lagt til! Takk

Reply:
Lagt de til nå.